Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt

```diff
@@ -416,16 +364,15 @@ internal class MonthView @JvmOverloads constructor(
     day: LocalDate,
     range: CalendarRange,
     paddingX: Int,
-    paddingY: Int,
     x: Int,
     y: Int
   ) {
-    rangeBackPaint.color = if (isColoredCalendar()) rangeColorColored else rangeColorNonColored
+    rangeBackPaint.color = rangeColorNonColored
```

Comment:
It seems like rangeBackPaint always have `rangeColorNonColored` colour. This assignments may be removed now.